src/app/components/elements/Author/index.jsx

```diff
@@ -5,7 +5,7 @@ import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
 import Icon from 'app/components/elements/Icon';
 import { Link } from 'react-router';
 import { authorNameAndRep } from 'app/utils/ComponentFormatters';
-import AuthorDropdown from './AuthorDropdown';
+import AuthorDropdown from '../AuthorDropdown';
```

Comment:
since AuthorDropdown is only used in this component would it make sense to colocate it in the Author component's subdirectory?

Reply:
I'm not so sure, as a component's usefulness is partially prospective - that it *can* be used elsewhere... and one day might be...

Reply:
makes sense yeah; let's just leave it